### ISSUE TITLE
Does not use new Async Roll system, using deprecated Sync Rolls instead

### ISSUE THREAD

**Issue by flamewave000** [RedReign/FoundryVTT-BetterRolls5e#284]
I'm the dev for [DF Manual Rolls](https://github.com/flamewave000/dragonflagon-fvtt/tree/master/df-manual-rolls) and part of my migration work tonight to FoundryVTT 0.8.6 has been to integrate with the awesome new Async Rolls system. Right now, BetterRolls appears to be using the deprecated Synchronous rolls. This means it will be completely incompatible with not only my own module, but others that work within the new Async Roll system.

**Comment by CarlosFdez** [comment]
To be honest, I've been holding off on async rolls specifically because I'm waiting for Pixel Dice to release a simulator for their rolling system (I've directly sent them a message to confirm that yes it will be a thing). The problem is that currently in better rolls the rolls are split up to allow multiple damage rows (even attack roll with advantage doesn't use 2d20kh, its two separate rolls) so supporting async rolls will need a very large refactor (few days minimum).

Knowing about DF Manual Rolls does increase what I can use to test an async roll refactor. If there are any more things that take advantage of async rolls I can add that to my list of test software and can maybe start sooner than a Pixel Dice simulator.

**Comment by flamewave000** [comment]
Yeah I think I remember when I made the first version of Manual Rolls for 0.7.9 that BetterRolls was making multiple Roll objects.
